api-platform/core/3.0/config/packages/api_platform.yaml

```diff
@@ -0,0 +1,11 @@
+api_platform:
+    title: Hello API Platform
+    version: 1.0.0
+    # Mercure integration, remove if unwanted
+    mercure: ~
```

Comment:
Shouldn't this rely on a smart default based on the presence of mercure in the dependencies, as done by FrameworkBundle for many settings ? If yes, it would allow not having that key in the recipe